client/modules/accounts/templates/dropdown/dropdown.html

```diff
@@ -78,3 +78,24 @@
     </a>
   </li>
 </template>
+
+
+<template name="staticPagesNav">
+  <div class="dropdown" role="menu">
+    <div class="dropdown-toggle" data-toggle="dropdown" data-hover="dropdown" data-delay="1000">
+      <span id="logged-in-display-name"> Pages <b class="caret"></b></span>
+    </div>
+    <div class="dropdown-menu static-pages-dropdown" role="menu">
+      {{#each staticPage in staticPages}}
+      <li class="dropdown-apps-icon">
+        <a href="/pages/{{staticPage.slug}}" id="{{staticPage.title}}" title="{{staticPage.title}}">
+        <i class=""></i>
```

Comment:
Yes, this list the number of pages in the dropdown menu on the navigation bar. It increases as more pages are being created.